src/lib/containers/access_requirement_list/AccessRequirementList.tsx

```diff
@@ -84,31 +109,43 @@ export default function AccessRequirementList({
             Number(requirementB.isApproved) - Number(requirementA.isApproved)
           )
         })
-        .map(status => {
+        .map((status) => {
           return parseInt(status.accessRequirementId)
         })
 
-      requirements = requirements.sort((requirementA, requirementB) => {
-        return (
-          sortOrder.indexOf(requirementA.id) -
-          sortOrder.indexOf(requirementB.id)
-        )
-      })
+      const requirementsAndStatuses = requirements
```

Comment:
just an FYI lodash has a [sortBy](https://lodash.com/docs/4.17.15#sortBy) and [orderBy](https://lodash.com/docs/4.17.15#orderBy) which might make things like that easier